build.gradle

```diff
@@ -143,21 +147,41 @@ test {
     systemProperty 'tests.security.manager', 'false'
 }
 
+File repo = file("$buildDir/testclusters/repo")
+def _numNodes = findProperty('numNodes') as Integer ?: 1
 testClusters.integTest {
     testDistribution = "OSS"
-    if (System.getProperty("cluster.debug") != null) {
-        jvmArgs('-agentlib:jdwp=transport=dt_socket,server=n,suspend=y,address=5005')
+    // Cluster shrink exception thrown if we try to set numberOfNodes to 1, so only apply if > 1
+    if (_numNodes > 1) numberOfNodes = _numNodes
+    // When running integration tests it doesn't forward the --debug-jvm to the cluster anymore
+    // i.e. we have to use a custom property to flag when we want to debug elasticsearch JVM
+    // since we also support multi node integration tests we increase debugPort per node
+    if (System.getProperty("es.debug") != null) {
```

Comment:
may be helpful to update the README corresponding part 😁

Reply:
Accidental change as I was using code from months ago which had a different cluster debug property. Changed back to cluster.debug for cluster JVM.